invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitiesList.js

```diff
@@ -68,35 +68,43 @@ export class RecordCommunitiesList extends Component {
   };
 
   render() {
-    const { recordCommunitySearchEndpoint } = this.props;
+    const { recordCommunitySearchEndpoint, permissions } = this.props;
     const { communities, loading, error } = this.state;
     const communityItems = communities?.slice(0, 2).map((community) => {
       return (
         <Item key={community.id}>
           <Image size="mini" src={community.links.logo} />
           <Item.Content verticalAlign="middle">
-            <Item.Header
-              as="a"
-              href={community.links.self_html}
-              className="ui small header"
-            >
+            <Item.Header as="a" size="small" href={community.links.self_html}>
               {community.metadata.title}
             </Item.Header>
           </Item.Content>
         </Item>
       );
     });
 
+    if (loading) {
+      return (
```

Comment:
Minor: there are at least 4 `return` in this method. Can we refactor to improve readability?
Warning: I have touched this in [my PR too](https://github.com/inveniosoftware/invenio-app-rdm/pull/2079) :(, in particular to refactor those `2` and `3` numbers around.

Reply:
no worries, I will wait for yours to be merged and then rebase